src/locales/tr.json

```diff
@@ -1,8 +1,179 @@
-  "footer": {
-    "termsOfService": "Kullanım Şartları",
-    "contact": "İletişim",
-    "about": "Hakkımızda",
-    "recentPosts": "Güncel Paylaşımlar"
+  "successStories": {
+    "header": "Başarı Hikayeleri",
+    "paragraph": "Şimdiye kadar 1000 fazla kişiye mutluluk ulaştırmayı başardık. Bu sayının çoğalmasında bize yardım edin ve dünyanın değişimine katkıda bulunun"
+  },
+  "whoWeAre": {
+    "header": "Misyonumuz",
+    "paragraph": "Bağış yapmaya istekli kişiler ile bunlara ihtiyacı olan kişiler arasındaki bağlantıyı sağlamak ve bağışı en çok ihtiyacı olan kişilere ulaştırmak.",
+    "howItWorks": "Nasıl Çalışır?"
+  },
+  "contact": {
+    "aletMessage": "Mesajınınz başarıyla gönderildi",
+    "formName": "Adı ve Soyadı",
+    "formNamePlaceholder": "Adı ve Soyadı",
+    "formEmail": "Email",
+    "formEmailPlaceholder": "örnek@gmail.com",
+    "formMessagePlaceholder": "Lütfen mesajınızı yazınız",
+    "formButton": "Gönder",
+    "header": "Bize Ulaşın",
+    "phone": "0090 536 437 48 43",
+    "email": "bizeulasın@fortunefound.org",
+    "par1": "Sorularınızın cevaplarını almak için lütfen bize ulaşın"
+  },
+  "loginPages": {
+    "login": {
+      "logIn": "Giriş Yap",
+      "email": "Email",
+      "password": "Şifre",
+      "logInBtn": "Giriş Yap",
+      "forgotPass": "Şifremi unuttum",
+      "needAcc": "Bir hesaba mı ihtiyacın var?",
+      "signUp": "Hesap Oluştur",
+      "logInGoogle": "Google ile giriş yap",
+      "logInFacebook": "Facebook ile giriş yap",
+      "alertFailLogin": "Giriş yapılamadı.",
+      "alertLoginSuccess": "Hesap başarıyla oluşturuldu"
+    },
+    "signup": {
+      "signUp": "Hesap Oluştur",
+      "name": "İsim",
+      "surname": "Soyisim",
+      "email": "Email",
+      "password": "Şifre",
+      "passwordConfirm": "Şifre Doğrulama",
+      "getStarted": "Başla",
+      "haveAnAccount": "Bir hesabım var",
+      "logIn": "Giriş Yap",
+      "passNotMatch": "Şifreler Eşleştirilemedi",
+      "failCreateAcc": "Hesap oluşturulamadı"
+    },
+    "forgotPassword": {
+      "passReset": "Şifre yenileme",
+      "email": "Email",
+      "resetBtn": "Şifre yenileme",
+      "login": "Giriş Yap",
+      "needAccount": "Bir hesaba mı ihtiyacın var?",
+      "signup": "Hesap Oluştur",
+      "alertMessage": "Gelen kutunu kontrol et!",
+      "alertError": "Şifre yenilenemedi"
+    },
+    "faceLogin": {
+      "welcome": "Hoşgeldiniz"
+    },
+    "logout": {
+      "alertSuccess": "Başaryıla Çıkış Yapıldı",
+      "alertLogoutFail": "Çıkış Yapılamadı",
+      "signout": "Çıkış yap",
+      "updateProfile": "Profilini yenile",
+      "logOut": "Çıkış yap"
+    },
+    "updateLoginInfo": {
+      "passNotMatch": "Şifreler Eşleştirilemedi",
+      "failUpdate": "Profil yenilenemedi",
+      "updateProfile": "Profilini yenile",
+      "email": "Email",
+      "password": "Şifre",
+      "passConfirm": "Şifre Doğrulama",
+      "update": "Güncelle",
+      "cancel": "İptal"
+    }
+  },
+  "about": {
```

Comment:
I think there are some updated stuff here if you look at the English file you’ll see them